include/sentry.h

```diff
@@ -1050,6 +1063,39 @@ SENTRY_API void sentry_remove_transaction(void);
  */
 SENTRY_API void sentry_set_level(sentry_level_t level);
 
+/**
+ * Adds a new attachment to be sent along.
+ *
+ * `path` is assumed to be in platform-specific filesystem path encoding.
+ * API Users on windows are encouraged to use `sentry_add_attachmentw` instead.
+ *
+ * See the NOTE on attachments above for restrictions of this API.
+ */
```

Comment:
Does it make sense to explicitly say something about the ownership of `*path`?  (same for remove attachment)